### PR TITLE
add additional maintainers

### PR DESCRIPTION
**Pull request by jreback**

### CODE REVIEW

**Comment by jreback** [comment]
@mvanniekerk or @jakirkham if you can merge would be great.

**Comment by jreback** [comment]
cc @davidastephens

**Comment by jreback** [comment]
@femtotrader as well

File: recipe/meta.yaml [hunk 1]
```diff
@@ -48,3 +48,5 @@ extra:
   recipe-maintainers:
     - mvanniekerk
```
Comment (jakirkham) [inline recipe/meta.yaml]: I think this is suppose to be, @mariusvniekerk . Would you mind correcting this as well?

xref: https://github.com/conda-forge/staged-recipes/pull/2834#discussion_r117000109

Comment (mariusvniekerk) [reply recipe/meta.yaml]: yeah i somehow flubbed my own username

**Comment by jreback** [comment]
ok fixed up and corrected @mariusvniekerk name

**Comment by mariusvniekerk** [comment]
I think we need to have the 0.4.0 one merged before this one.  Seems something in pandas changed.

**Comment by jakirkham** [comment]
Honestly we don't need CI for maintainer changes like this. I'd still be ok merging so we can get everybody on-board. That way, everyone here can review and merge the update.

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: np. Honestly I copy mine from my profile just to avoid potential issues like this.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -46,5 +46,7 @@ about:
 
 extra:
   recipe-maintainers:
-    - mvanniekerk
+    - mariusvniekerk
     - femtotrader
+    - jreback
+    - davidastephens
```
Comment (jakirkham) [inline recipe/meta.yaml:52]: Is it ok for you to be a maintainer here, @davidastephens?

**Comment by davidastephens** [comment]
@jakirkham yes no problem.

**Comment by femtotrader** [comment]
I was suggesting this https://github.com/conda-forge/staged-recipes/pull/2834#pullrequestreview-33075860
I'm very busy these last months... and so I don't want to be a retarder for new releases.

**Comment by jakirkham** [comment]
Have added everyone to this team manually as the team update script is a bit of a bear ATM. Maybe this is more incentive to look at revamping it. ( https://github.com/conda-forge/conda-forge-webservices/issues/63 )

**Pull request merged**